### DISCUSSION TITLE
BTicino K4003C blue LED support

### DISCUSSION THREAD

**Discussion by diego065** [Koenkk/zigbee2mqtt#6352] [Devices]
Hello! I start to use the bticino switch (K4003C) with zigbee2mqtt and it’s working perfectly… The only thing that I missed is the use of the small blue led even if the switch is off. I saw that someone is using it with the “custom” devices.js but I am using the hassos plugin and I have no idea on how to integrate it for my system!!
Does anyone can help me?
Thanks

**Comment by sakara** [comment]
Send MQTT messages manually using the “MQTT” Integration in the frontend of HA. Choose the “Configuration” tab, click “Integrations” and click the “Configure” option under the “MQTT” integration. Enter something similar to the example below into the “topic” field under “Publish a packet".

zigbee2mqtt/FRIENDLY_NAME/set

FRIENDLY_NAME : is the friendly name of you K4003C

and use this payload:

{
“permanent_led”: “ON”,
“led_when_on”: “ON”
}

permanent_led : enable or disable the permanent blue LED. Values: ON / OFF (default)
led_when_on : enables the LED when the light is on. Values: ON / OFF (default)
